core/io/resource_format_binary.cpp

```diff
@@ -2142,7 +2142,7 @@ static String _resource_get_class(Ref<Resource> p_resource) {
 	}
 }
 
-Error ResourceFormatSaverBinaryInstance::save(const String &p_path, const Ref<Resource> &p_resource, uint32_t p_flags) {
+Error ResourceFormatSaverBinaryInstance::save(const String &p_path, const Ref<Resource> &p_resource, uint32_t p_flags, const int p_integrity_level) {
```

Comment:
```suggestion
Error ResourceFormatSaverBinaryInstance::save(const String &p_path, const Ref<Resource> &p_resource, uint32_t p_flags, int p_integrity_level) {
```

I see you're adding `const` to some `int` parameters but not to others. My take is the same as always, especially for parameters, it makes readability worse. In any case, if there was consensus to use `const` everywhere it fits, let's do a whole-codebase pass. I'd therefore suggest not adding them in this PR.